src/libs/TypingMiddleware.js

```diff
@@ -22,12 +25,17 @@ export default function typingMiddleware() {
             message.nick :
             message.params[0];
 
+        // if its a privmsg without typing tag emit done
+        let status = (message.tags['+draft/typing']) ?
```

Comment:
This would be simpler read as just `let status = message.tags['+draft/typing'] || 'done';`